superflore/generators/ebuild/run.py

```diff
@@ -106,6 +106,7 @@ def main():
         sys.exit(1)
     elif args.pr_only and not args.output_repository_path:
         err('Invalid args! no repository specified')
+        sys.exit(1)
```

Comment:
Use parser.error to both send an error message and exit. https://docs.python.org/2/library/argparse.html#exiting-methods It will also give the usage again.

This would be good for the other error above